aodndata/templates/soop_co2_nrt_nc_template.json

```diff
@@ -0,0 +1,221 @@
+      "standard_name": "sea_surface_salinity",
+      "reference_datum": "sea surface",
+      "units": "1e-3",
+      "valid_max": 42.0,
+      "valid_min": 0.0,
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    },
+    "ATMP_raw":{
+      "_FillValue": -999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "long_name": "atmospheric pressure",
+      "standard_name": "air_pressure",
+      "reference_datum": "sea surface",
+      "units": "hPa",
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    },
+    "WSPD_raw":{
+      "_FillValue": -999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "long_name": "wind speed",
+      "standard_name": "wind_speed",
+      "reference_datum": "sea surface",
+      "units": "m s-1",
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    },
+    "WDIR_raw":{
+      "_FillValue": -999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "long_name": "wind direction",
+      "standard_name": "wind_from_direction",
+      "reference_datum": "magnetic north",
+      "positive": "clockwise",
+      "comment": "true wind direction where 0 is North and 90 is East",
+      "units": "degree",
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    },
+    "Tsg_flow_raw":{
+      "_FillValue": -999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "long_name": "thermosalinograph water flow",
+      "reference_datum": "sea surface",
+      "units": "l min-1",
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    },
+    "TEMP_Tsg_raw":{
+      "_FillValue": -999.0,
+      "_datatype": "float64",
+      "_dimensions": ["TIME"],
+      "long_name": "Sea water temperature measured by thermosalinograph",
+      "standard_name": "sea_surface_temperature",
+      "reference_datum": "sea surface",
+      "units": "degree_Celsius",
+      "valid_max": 40.0,
+      "valid_min": -2.0,
+      "coordinates": "TIME LATITUDE LONGITUDE"
+    }
+  },
+  "project": "Integrated Marine Observing System (IMOS)",
+  "Conventions": "CF-1.6,IMOS-1.4",
+  "source": "ship observation",
+  "keywords": "Oceans>Ocean Temperature>Sea Water Temperature ;Oceans>Salinity/Density>Salinity;Oceans>Oceans Optics>Fluorescence ;Oceans>Ocean Optics>Turbidity; Oceans>Ocean Chemistry>Pigments>Chlorophyll",
+  "references": "http://www.imos.org.au",
+  "naming_authority": "IMOS",
+  "file_version": "Level 0 - Raw data",
+  "acknowledgement": "Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: \"Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is a national collaborative research infrastructure, supported by the Australian Government. This data was collected by the Environmental Protection Authority (EPA) of Victoria. Assistance with logistical and technical support for this project has been provided by the Spirit of Tasmania 1 vessel operator, TT lines.\"",
```

Comment:
the acknowledgement should be:  "Any users of IMOS data are required to clearly acknowledge the source of the material derived from IMOS in the format: \"Data was sourced from the Integrated Marine Observing System (IMOS) - IMOS is a national collaborative research infrastructure, supported by the Australian Government."